src/transactions/TransactionUtils.cpp

```diff
@@ -1013,6 +1013,17 @@ getAsset(AccountID const& issuer, AssetCode const& assetCode)
     return asset;
 }
 
+bool
+claimableBalanceFlagIsValid(ClaimableBalanceEntry const& cb)
+{
+    if (cb.ext.v() == 1)
+    {
+        return (cb.ext.v1().flags & ~MASK_CLAIMABLE_BALANCE_FLAGS) == 0;
```

Comment:
in code you also assume (in `setClaimableBalanceClawbackEnabled`) that if `ext.v() == 1`, then `ext.v1().flags != 0` ; ie can't clear the flag anywhere in code. So this is really `cb.ext.v1().flags == MASK_CLAIMABLE_BALANCE_FLAGS`

Reply:
Good point. Done.